group_vars/all.yml.dist

```diff
@@ -59,6 +60,37 @@ ansible_nas_user: david
 # Your email address, used for LetsEncrypt SSL certs
 ansible_nas_email: me@email.com
 
+###
+### Watchtower
+###
+# Sets the 6 field cron schedule to use for checks and updates
+# Keep in mind that this will stop and start containers if there is an update
+# So probably best not to do it every 10 minutes unless you want your plex stream
+# interrupted
+# Default below is every day at 5am (local time if you have set
+# your local timezone above)
+watchtower_cron_schedule: "0 0 5 * * *"
```

Comment:
All of this cron stuff can be moved down to the "you won't need to edit this" section. A 5am nightly update is sufficient for 99% of use cases IMO